packages/api/src/configuration/exceptions/exception.abstract.ts

```diff
@@ -0,0 +1,15 @@
+export abstract class IFormatExceptionMessage {
```

Comment:
@soumyadip007 nest kindha follows angular pattern hence the exception should be ether in their respective files / controllers or insider services

Reply:
Yes I've already moved that.